### PR TITLE
FPL-364: Change jurisdiction and case type names

### PR DESCRIPTION
**Pull request by mh-hmcts**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: ccd-definition/CaseType.json [hunk 1]
```diff
@@ -2,8 +2,8 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "Shared_Storage_DRAFTType",
```
Comment (geek-soft) [inline ccd-definition/CaseType.json]: Please change case ID

Comment (geek-soft) [reply ccd-definition/CaseType.json]: If not deployment to production will override current shared storage POC.

Comment (mh-hmcts) [reply ccd-definition/CaseType.json]: agreed planned and raised raised as separate task

File: ccd-definition/Jurisdiction.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "PUBLICLAW",
```
Comment (geek-soft) [inline ccd-definition/Jurisdiction.json]: I guess that ID can remain unchanged. New case type will just appear under jurisdiction that is in use. I guess having rename only is sensible.

File: ccd-definition/CaseType.json (hunk 1 above)
Comment (geek-soft) [inline ccd-definition/CaseType.json]: Not sure what is best name for the ID, I guess we could `CARE_SUPERVISION_EPO`.

**Pull request merged**